docs/minio-admin-complete-guide.md

```diff
@@ -0,0 +1,311 @@
+# Minio Admin Complete Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
```

Comment:
Shouldn't this be `MC Admin Complete Guide`

Reply:
I am not sure `MC` didn't sound right `Minio Client Admin` also doesn't sound right. So `Minio Admin` was only that made since it is `Minio Admin` more so than `mc admin`

Reply:
The reason to avoid `mc` is that it doesn't make sense to call this 'client admin' since it is simply `admin`. I will prefer to keep it `Minio Admin` specifically because it is a `Minio Admin` guide not a client guide.